src/i18n/es.json

```diff
@@ -44,6 +44,7 @@
   "post.myTimeline": "Mi Línea del tiempo",
   "post.wasLive": "estuvo en vivo",
   "post.copyPath": "Copiar camino",
+  "post.openInNewTab": "Open in new tab",
```

Comment:
Does not look spanish :D

Reply:
Coach ops told me I can leave anything that's only coach facing in English, so I made this coach facing only even if it would be useful for users, just so I wouldn't have to translate it :D